snapshots/health/2023-08-11/unaids_hiv_children.xlsx.dvc

```diff
@@ -0,0 +1,19 @@
+meta:
+  name: Global AIDS Update (UNAIDS, 2023)
+  publication_date: '2023'
+  source_name: UNAIDS
+  source_published_by: 'The path that ends AIDS: UNAIDS Global AIDS Update 2023. Geneva:
+    Joint United Nations Programme on HIV/AIDS; 2023. Licence: CC BY-NC-SA 3.0 IGO.'
+  url: https://aidsinfo.unaids.org/
+  source_data_url:
+  license_url: https://www.unaids.org/en/terms-of-use
+  license_name: UNAIDS terms
+  date_accessed: 2023-08-11
+  is_public: true
+  description: |
+    This report makes clear that there is a path to end AIDS. Taking that path will help ensure preparedness to address other pandemic challenges, and advance progress across the Sustainable Development Goals. The data and real-world examples in the report make it very clear what that path is. It is not a mystery. It is a choice. Some leaders are already following the path—and succeeding. It is inspiring to note that Botswana, Eswatini, Rwanda, the United Republic of Tanzania and Zimbabwe have already achieved the 95–95–95 targets, and at least 16 other countries (including eight in sub-Saharan Africa) are close to doing so.
```

Comment:
This description seems a little odd - it's not really a description of the dataset but more of a statement about AIDS.